### PR TITLE
fix(FEC-9013): [iOS] - Player doesnt play ads native when playsinline is false

### PR DESCRIPTION
**Pull request by Yuvalke**
### Description of the Changes

force delay until video tag will be loaded and then load the ima, ima ads take the inline configure from our video tag 

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/ima.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
    */
   static defaultConfig: Object = {
     debug: false,
-    delayInitUntilSourceSelected: false,
+    delayInitUntilSourceSelected: true,
```
Comment (yairans) [inline src/ima.js]: should be true for ios only

File: src/ima.js [hunk 2]
```diff
@@ -454,6 +455,20 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
...
 
+  /**
+   * set default Configuration for plugin
+   * @private
+   * @returns {void}
+   * @instance
+   * @memberof Ima
+   * @param {Object} config - configuration of app for ima plugin
+   */
+  _setDefaultConfiguration(config: Object): void {
```
Comment (OrenMe) [inline src/ima.js]: why do you need config if you are using `this` to access it anyway, seems redundant.

File: src/ima.js [hunk 3]
```diff
@@ -454,6 +455,20 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
...
+    //check if delayInitUntilSourceSelected set in app configuration, else set true for ios
+    this.config.delayInitUntilSourceSelected =
```
Comment (OrenMe) [inline src/ima.js]: I think a simple if is more readable here.
```javascript
if (typeof this.config.delayInitUntilSourceSelected !== `boolean`) {
  this.config.delayInitUntilSourceSelected = this.player.env.os.name === 'iOS'
}
```

File: src/ima.js (hunk 2 above)
Comment (Yuvalke) [reply src/ima.js]: It's not, in super method there is merge of all configurations.
I'm looking for the application configuration, this.config is the configuration after merge with default configuration so it always be false after merge with default(false).
So I'm taking the application config and check there assignment.

File: src/ima.js (hunk 1 above)
Comment (Yuvalke) [reply src/ima.js]: Added in _setDefaultConfiguration method

**Pull request merged**